### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by Orangepepermint** [msharrock/deepbleed#10]
Hi, thanks for sharing your code.
I'm try to train my one datasets with your model. But I didn't find the training script. can you also share that?
That will be a great help!

Thanks in advance

**Comment by msharrock** [comment]
Hi, its really simple to train this model in tf keras. Simply load your dataset using tensorflow data types, import the model, compile and select optimizer and metric and then model.train(dataset). Very simple process.  see https://www.tensorflow.org/guide/keras/train_and_evaluate
